src/features/home-todo/ui/style.css.ts

```diff
@@ -0,0 +1,70 @@
+import { style } from '@vanilla-extract/css';
+import { font } from 'shared/styles/font.css';
+import theme from 'shared/styles/theme.css';
+
+export const ToDoListContainer = style({
+  padding: '8px 12px',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  gap: '8px',
+  borderRadius: '12px',
+  alignSelf: 'stretch',
+  backgroundColor: theme.white,
+});
+
+export const ToDoListHeader = style({
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'ceter',
+  justifyContent: 'space-between',
+});
+
+export const ToDoListTitle = style({
+  color: theme.gray[600],
+  fontFamily: 'Pretendard',
+  fontSize: font.H5.fontSize,
+  fontWeight: font.H5.fontWeight,
+  lineHeight: font.H5.lineHeight,
+  fontStyle: 'normal',
+});
```

Comment:
다음과 같이 작성하여 코드의 길이를 줄일 수 있습니다.

```suggestion
export const ToDoListTitle = style({
  color: theme.gray[600],
  ...font.H5
});
```

Reply:
다른 코드에도 반영해 주세요.